src/assets-list/assets-list.js

```diff
@@ -0,0 +1,91 @@
+import jQuery from 'jquery';
+import template from './assets-list.pug';
+const assetUrl= 'http://fuego-app.herokuapp.com/api/assets';
+import Assets from '../services/assets';
+
+const component = jQuery('<div class="assets-list-component component"></div>');
+//console.log(component);
+
+// jQuery.getJSON(assetsURL).then(function(response) {
```

Comment:
never leave commented code since this makes maintaining the file more difficult:

- Other devs won't be sure if they should remove this code
- It makes the file file bigger (= more lines to read)

Reply:
If you want to save the previous functionality, when you delete this line just commit that change and explain why it was removed.

ex: "removed JSON request since static assets are going to be used"

or, in this case: "refactored the assets loading function"